lib/api/src/identity/extensions/Identity.ts

```diff
@@ -0,0 +1,20 @@
+import { Identity as AbstractIdentity } from '../Identity'
+
+export class Identity extends AbstractIdentity {
+
+  /**
+   * {@inheritdoc}
+   */
+  public getRedirectUrl (path: string) : string {
+    return `${browser.identity.getRedirectURL()}${path}`
```

Comment:
does this handle slashes the same way chrome does?

Reply:
OK, this is odd - documentation specifies no params for this method ([here](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/identity/getRedirectURL)), so I assumed we would have to implement appending `path` via interpolation/concatenation ourselves. I tested to confirm it works, which it does - see output from Firefox console below:
```js
path = 'oauth2'
"oauth2"
`${browser.identity.getRedirectURL()}${path}`
"https://c5d72721b35687d2406e43f626cdd2d35852ff48.extensions.allizom.org/oauth2"
```
Then I thought I would just check if method will accept a path param and it does:
```
browser.identity.getRedirectURL()
"https://c5d72721b35687d2406e43f626cdd2d35852ff48.extensions.allizom.org/"
browser.identity.getRedirectURL('oauth2')
"https://c5d72721b35687d2406e43f626cdd2d35852ff48.extensions.allizom.org/oauth2"
```
Would you prefer we keep current template string implementation or use as identical to `chrome.identity.getRedirectURL()` by passing path param direct to browser API??

Reply:
As long as we document it on our side, either option is fine if it has the same behaviour I'd say.

Reply:
OK I am leaving it as is because otherwise the defs are gonna complain and I don't want to have to force a type assertion on it - lets implement as consistent with the available docs